checkov/common/bridgecrew/wrapper.py

```diff
@@ -37,6 +39,27 @@ def _put_json_object(s3_client: BaseClient, json_obj: Any, bucket: str, object_p
         raise
 
 
+def _get_json_object(
+    s3_client: BaseClient, bucket: str, object_path: str, throw_json_error: bool = True
+) -> dict[str, Any] | None:
+    try:
+        result_body = s3_client.get_object(Bucket=bucket, Key=object_path)['Body'].read().decode('utf-8')
+        return cast("dict[str, Any]", json.loads(result_body))
+    except ClientError as e:
+        logging.warning(e)
+    except JSONDecodeError as e:
+        if throw_json_error:
+            logging.error(e)
+            raise e
+        else:
+            logging.warning(f'Failed to get json object due to {str(e)}')
+    except Exception as e:
+        logging.error(e)
+        raise e
```

Comment:
```suggestion
    except ClientError as e:
        logging.warning(e)
    except JSONDecodeError as e:
        if throw_json_error:
            logging.error(e)
            raise e
        else:
            logging.warning(f'Failed to get json object due to {str(e)}')
    except Exception as e:
        logging.error(e)
        raise e
```
for all the `logging` usage can you add some message and then use `exc_info=True` to automatically log the error message + stack trace